README.md

```diff
@@ -7,3 +7,6 @@ Config files for [Plex Meta Manager](https://metamanager.wiki/) following Spanis
 | Set                          | Author                                  | Link                               |
 | ---------------------------- | --------------------------------------- | ---------------------------------- |
 | Alien and Predator Universes | [AMC](https://theposterdb.com/user/AMC) | https://theposterdb.com/set/109057 |
+| ---------------------------- | --------------------------------------- | ---------------------------------- |
```

Comment:
Same as in Rambo. No need to include `|----|` lines 10 and 12

Reply:
Fixed